src/components/MewSwitch/MewSwitch.vue

```diff
@@ -1,15 +1,14 @@
 <template>
   <div class="d-flex align-center"> 
     <span
-      @click="onToggle()"
```

Comment:
hmm I think they want the label to toggle the switch ,  unless jacks design say otherwise!